config/router-chatbot.js

```diff
@@ -7,6 +7,14 @@ const express = require('express');
 const router = express.Router(); // eslint-disable-line new-cap
 const mobilecommons = rootRequire('lib/mobilecommons');
 const logger = app.locals.logger;
+const Promise = require('bluebird');
+
+function CampaignClosedError() {}
```

Comment:
@sergii-tkachenko Where would you suggest placing the extensions file? into `config` at the same directory? New folder?

Related question -- Is `config` the best place for this `router-chatbot.js` lives? I've following loosely following Sails conventions in attempt to make things easy to find.